mobile/src/main/java/org/openhab/habdroid/ui/preference/ItemUpdatingPreference.kt

```diff
@@ -206,6 +206,11 @@ fun String?.toItemUpdatePrefValue(): Pair<Boolean, String> {
     return Pair(this!!.substring(0, pos).toBoolean(), substring(pos + 1))
 }
 
+fun Any?.toItemUpdatePrefValue(): Pair<Boolean, String> {
+    @Suppress("UNCHECKED_CAST")
+    return this as Pair<Boolean, String>
+}
```

Comment:
How's this related to the issue in question?

(Besides that, I'm not a fan of this extension function: it's only used once and only used as a warning suppressor. Supressing the warning were only OK for me if the method ensures the cast is safe)

Reply:
This change is a left over from a previous attempt to fix this issue. Since it's only used in one place now, I'll remove it.